shared/js/template.js

```diff
@@ -114,24 +114,25 @@
    *                          HTML that is known and are
    *                          "known" to ignore.
    */
-  Template.prototype.interpolate = function(data, options) {
-    // This _should_ be rewritten to use Firefox's support for ES6
-    // default parameters:
-    // ... = function(data, options = { safe: [] }) {
-    //
-    options = options || {};
-    options.safe = options.safe || [];
```

Comment:
If we want to check if `options.safe` is correct, I'd prefer to throw if it's present but is not an array. No quiet ignore :)